src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/permissions/AgoraPermissions.scala

```diff
@@ -47,7 +47,7 @@ object AgoraPermissions {
 }
 
 case class AgoraPermissions(permissions: Int) {
-  def this(varPermissions: Int*) {
+  def this(varPermissions: Int*) = {
```

Comment:
Isn't `this` a protected keyword?

Reply:
Is this a weird alternative to writing an `apply` method? In which case... "could this be changed to an `apply` method"?

Reply:
It appears to be a type alias to construct the object

Finding usages takes me to
```
val authorization = new AgoraPermissions(Read, Write, Redact)
```

I chose not to mess with it, in the spirit of making the minimal possible changes.